debian/packetfence.preinst

```diff
@@ -64,6 +64,8 @@ case "$1" in
         echo "RateLimitInterval=0" >> /etc/systemd/journald.conf
         echo "RateLimitBurst=0" >> /etc/systemd/journald.conf
         echo "ForwardToWall=no" >> /etc/systemd/journald.conf
+        echo "SystemMaxUse=1G" >> /etc/systemd/journald.conf
```

Comment:
Unfortunately no.
Debian uses an older version of systemd than RHEL which does not allow overriding journald.conf in a separate file.
It sucks tremendously, I know.